src/PJ_cart.c

```diff
@@ -486,13 +486,13 @@ int pj_cart_selftest (void) {
     /* proj_info()                                                            */
     /* this one is difficult to test, since the output changes with the setup */
     info = proj_info();
-    if (info.version) {
+    if (info.version[0] != '\0' ) {
         char tmpstr[64];
         sprintf(tmpstr, "%d.%d.%d", info.major, info.minor, info.patch);
         if (strcmp(info.version, tmpstr)) return 55;
     }
```

Comment:
I see that if `info.version` is non-empty, it should match the expected version. But should it ever be empty?